tests/test_transforms/test_inference/test_transform.py

```diff
@@ -17,6 +17,8 @@
 from etna.transforms import DensityOutliersTransform
```

Comment:
We should also add these transforms to inference tests with `inverse_transform`.